### PR TITLE
Use "[argument]" instead of "<argument>?"

### PR DESCRIPTION
**Pull request by fxiqval**
The standards suggest this change:

### CODE REVIEW

**Comment by Jannyboy11** [comment]
Should be [<argument>] then, right? Because the fish brackets indicate it is a placeholder. '[argument]' would mean the literal "argument", optionally.

**Comment by fxiqval** [comment]
Yes, I forgot about the placeholder. The standards are here:
 http://courses.cms.caltech.edu/cs11/material/general/usage.html#:~:text=Optional%20arguments%20should%20always%20be,an%20argument%20is%20optional%2C%20always.

**Comment by Jannyboy11** [comment]
I think it's probably best to adhere to the usage string syntax that vanilla Minecraft itself uses as well: https://minecraft.fandom.com/wiki/Commands#Syntax (which coincidentally seems to be the same that the syntax that you're proposing).

**Comment by Jannyboy11** [comment]
You are missing the changes in the README.md by the way; the command usages are written there as well.

File: README.md [hunk 1]
```diff
@@ -42,8 +42,8 @@ Note that for integration with PerWorldInventory, `load-data-on-join` needs to b
 
 #### InvSee++_Give
 ##### Commands:
-- `/invgive <player> <item type> <amount>? <nbt tag>?`
-- `/endergive <player> <item type> <amount>? <nbt tag>?`
+- `/invgive <player> <item type> [amount] [nbt tag]`
```
Comment (Jannyboy11) [inline README.md:45]: should be `[<amount>]` and `[<nbt tag>]`

File: README.md [hunk 2]
```diff
@@ -42,8 +42,8 @@ Note that for integration with PerWorldInventory, `load-data-on-join` needs to b
 
 #### InvSee++_Give
 ##### Commands:
-- `/invgive <player> <item type> <amount>? <nbt tag>?`
-- `/endergive <player> <item type> <amount>? <nbt tag>?`
+- `/invgive <player> <item type> [amount] [nbt tag]`
+- `/endergive <player> <item type> [amount] [nbt tag]`
```
Comment (Jannyboy11) [inline README.md:46]: idem

File: README.md [hunk 3]
```diff
@@ -55,8 +55,8 @@ Note that for integration with PerWorldInventory, `load-data-on-join` needs to b
 
 #### InvSee++_Clear
 ##### Commands:
-- `/invclear <player> <item type>? <amount>?`
-- `/enderclear <player> <item type>? <amount>?`
+- `/invclear <player> [item type] [amount]`
```
Comment (Jannyboy11) [inline README.md:58]: `[<item type>]` `[<amount>]`

File: README.md [hunk 4]
```diff
@@ -55,8 +55,8 @@ Note that for integration with PerWorldInventory, `load-data-on-join` needs to b
 
 #### InvSee++_Clear
 ##### Commands:
-- `/invclear <player> <item type>? <amount>?`
-- `/enderclear <player> <item type>? <amount>?`
+- `/invclear <player> [item type] [amount]`
+- `/enderclear <player> [item type] [amount]`
```
Comment (Jannyboy11) [inline README.md:59]: idem

**Review by Jannyboy11 — CHANGES_REQUESTED** [review]
The [Minecraft wiki](https://minecraft.fandom.com/wiki/Commands#Syntax) states that angle brackets (`<>`) are used for to denote placeholders, and square brackets (`[]`) are used for optional arguments. Thus, to denote a placeholder for an optional argument, `[<placeholder>]` should be used (since `[foo]` would denote that the command would accept the literal "foo" as an optional argument).

**Comment by Jannyboy11** [comment]
Do you still plan to address my requested changes, or should I make the changes myself?

**Comment by fxiqval** [comment]
I will make the changes later today. So you're requesting `[optional]` or `[<optional>]` for optional arguments?

**Comment by Jannyboy11** [comment]
Either you didn't read my messages, or you are just trolling now. My earlier comments clearly state that it should be `[<argument>]` for optional arguments indicated with placeholders.

**Comment by Jannyboy11** [comment]
https://github.com/Jannyboy11/InvSee-plus-plus/commit/620c7c41c743ded67d3dec868d20c69acddc0ab0

**Pull request closed without merging**